model-customization/data-processing/seed-data-for-knowledge/seed-data-for-knowledge.ipynb

```diff
@@ -0,0 +1,377 @@
+    "contribution_summary = \"A Probabilistic Inference Approach to Inference-Time Scaling of Large Language Models (LLMs)\"\n",
+    "\n",
+    "# Add contribution information to the knowledge_contribution dictionary for it\n",
+    "knowledge_contribution = {\"name\": contribution_name, \"domain\": contribution_domain, \"summary\": contribution_summary}\n",
+    "contributions.append(knowledge_contribution)\n",
+    "\n",
+    "# NFL Rules Contribution\n",
+    "contribution2_name = \"nfl\"\n",
+    "contribution2_domain = \"sports rules\" \n",
+    "contribution2_summary = \"Official playing rules of the National Football League 2022, 2023\"\n",
+    "knowledge_contribution2 = {\"name\": contribution2_name, \"domain\": contribution2_domain, \"summary\": contribution2_summary}\n",
+    "contributions.append(knowledge_contribution2)\n",
+    "\n",
+    "for contribution in contributions:\n",
+    "    contribution_dir = WORKSPACE_DIR / contribution[\"name\"]\n",
+    "    contribution[\"dir\"] = contribution_dir"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2bd3b1bc-84b7-483c-910d-4983e94fe3c2",
+   "metadata": {},
+   "source": [
+    "To create contributions, define the `name` for the contribution, and the `domain` and `summary`. The `name`, `domain` and `summary` go into a dictionary called `knowledge_contribution` which gets added to a list called `contributions`.\n",
+    "\n",
+    "Once the list of `contributions` is set, a directory with each contribution name is created within the workspace and subdirectories for `source_documents`, `conversion`, `chunking`, `authoring` are created within the contribution name directory."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "109090af-c01d-494f-8dde-0daddcdb3cc8",
+   "metadata": {},
+   "source": [
+    "## Seed Data Creation\n",
+    "\n",
+    "To start the synthetic data generation process, users need to prepare a diverse set of questions and answers based off chunks from each source document. A chunk and question-and-answer pairs are called a seed example."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "26c41ed1-1d96-4388-9afe-ab40efd0c717",
+   "metadata": {},
+   "source": [
+    "### Install docling-sdg\n",
+    "\n",
+    "[Docling-sdg](https://github.com/docling-project/docling-sdg) project is used to generate question and answer pairs for seed examples."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4de1fd21-faab-4dbb-9217-191e02088678",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install -qq docling-sdg"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "81e33660-1541-4733-a2b8-4cd9a18ddff0",
+   "metadata": {},
+   "source": [
+    "### Initialize QA generator model & Number of Seed examples\n",
+    "\n",
+    "To generate seed examples you need to set: \n",
+    "1. The the Open AI compatible endpoint for the model generating question and answer pairs\n",
```

Comment:
In partcular, it can be a local endpoint. [`docling-sdg` README section](https://github.com/docling-project/docling-sdg?tab=readme-ov-file#generate) (which still itself does not specify that locally hosted Ollama models are supported).